docs/source/models.rst

```diff
@@ -499,3 +499,32 @@ Spark cluster and used to score the model.
 
     pyfunc_udf = mlflow.pyfunc.spark_udf(<path-to-model>)
     df = spark_df.withColumn("prediction", pyfunc_udf(<features>))
+
+The resulting UDF is based Spark's Pandas UDF and is currently limited to producing either a single
+value or an array of values of the same type per observation. You can control what result is
+returned by supplying ``result_type`` argument. Following values are supported:
+    * ``'int'`` or instanceof ``IntegerType``: The leftmost integer that can fit in
```

Comment:
Why would people ever pass an instanceof instead of passing IntegerType directly? It seems easier to just say IntegerType (and same with the other ones below). Also, you may want to link IntegerType to the PySpark page on IntegerType: https://spark.apache.org/docs/latest/api/python/pyspark.sql.html?highlight=integertype#pyspark.sql.types.IntegerType